src/resources/views/configuration/users/edit.blade.php

```diff
@@ -29,25 +31,124 @@
           </div>
 
         </div><!-- /.box-body -->
+      </form>
 
         <div class="box-footer">
 
+
           @if(auth()->user()->id != $user->id)
-            <a href="{{ route('configuration.users.edit.account_status', ['user_id' => $user->id]) }}"
-               class="btn btn-{{ $user->active ? 'warning' : 'success' }} pull-left">
+            <a type="button" class="btn btn-{{ $user->active ? 'warning' : 'success' }} pull-left" data-toggle="modal" data-target="{{ $user->active ? '#deactivateModal' : '#activateModal' }}">
+
               @if($user->active)
                 {{ trans('web::seat.deactivate_user') }}
               @else
                 {{ trans('web::seat.activate_user') }}
               @endif
             </a>
+
```

Comment:
These new sections need translations please.

Reply:
maybe we can do this in another PR or you add to this pull request, as i know that i won't match your wording?